### ISSUE TITLE
Display statistics using morris.js

### ISSUE THREAD

**Issue by flxai** [freifunkks/moinmoin-theme#2]
Use [morris.js](https://morrisjs.github.io/morris.js/lines.html) to render statistics for a cleaner look. Alternatively find something that works server-side and render static HTML that works with CSS (hovering) and without JS.

**Comment by jplatte** [comment]
Why do we need to generate HTML? Can't we just generate a new image file when there is new data instead of generating a new text file with info for a plugin?

For image generation, I would guess gnuplot should be an option (though I've never used it).

**Comment by flxai** [comment]
Yes, this would work as well. My assumption was, that interaction would lead to more information. For example being able to make out values at spikes faster by hovering, than having to look at the left.

We are currently using [RRDtool](http://oss.oetiker.ch/rrdtool/), which comes with its own (round robin) database and thus has a small footprint. But I think that throwing away this kind of anonymous statistical data over time is a bad idea.

**Comment by jplatte** [comment]
Well yes, interactivity is of course nice to have as well. I'd say it's up to the person who implements it. The perfect solution IMO would be having both, though this is only really a viable option if we manage to make both look the same.

About data persistence: I am with you on that, we should archive the stats in some form :)
